model/metadata.go

```diff
@@ -29,6 +29,7 @@ type Metadata struct {
 	UserAgent UserAgent
 	Client    Client
 	Cloud     Cloud
+	Message   Message
```

Comment:
Our "Metadata" is roughly equivalent to OpenTelemetry's "Resource" - information about the service, or the environment in which the service is operating. Message details don't belong here, as they're operation-specific.